lib/omnibus/software.rb

```diff
@@ -767,6 +746,19 @@ def overridden?
     # @!endgroup
     # --------------------------------------------------
 
+    #
+    # Path to where any source is extracted to.
+    #
+    # Files in a source directory are staged underneath here. Files from
+    # a url are fetched and extracted here. Look outside this directory
+    # at your own peril.
+    #
+    # @return [String] the full absolute path to the package root directory.
+    #
+    def package_dir
```

Comment:
Woah eventual consistency model....

Ok..  so would someone prefer extract_dir/source_staging_dir instead?  It's still the same regex and I honestly want to pick the name y'all like best since you're the ones who have to deal with it and I've already spend 4-5 days being bewildered with the code and don't want to inflict the same on you.